backend/src/main/java/com/llm_service/llm_service/service/ConversationService.java

```diff
@@ -11,13 +14,20 @@
 @RequiredArgsConstructor
 public class ConversationService {
     private final LLMService llmService;
+    private final ConversationPersistenceManager conversationPersistenceManager;
 
     public Conversation start() {
-        return null;
+        Conversation conversation =
```

Comment:
maybe text should be passed by parameter, and we should rename it to `title`  in all conversation related stuff.

Reply:
?